package.json

```diff
@@ -34,6 +34,8 @@
     "react": "^0.13.0"
   },
   "dependencies": {
+    "prop-types": "^15.5.0",
+    "react": "^15.5.0",
```

Comment:
Pretty sure we wouldn't want to pull in our own copy of `react` here. Perhaps as a peer dependency?